vkcs/networking/resource_port.go

```diff
@@ -105,6 +105,14 @@ func ResourceNetworkingPort() *schema.Resource {
 				Description: "(Conflicts with `security_group_ids`) If set to `true`, then no security groups are applied to the port. If set to `false` and no `security_group_ids` are specified, then the port will yield to the default behavior of the Networking service, which is to usually apply the \"default\" security group.",
 			},
 
+			"control_security_group_ids": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    false,
+				Default:     false,
+				Description: "If this argument is set to true, then the user has full control over `security_group_ids`.",
```

Comment:
Not user but thr resource. Also we need to specify behavior w/ default group.

Reply:
... and difference between empty set and no specified argument

Reply:
change to full_security_group_control